flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/utils/TableTestBase.scala

```diff
@@ -306,28 +301,24 @@ abstract class TableTestUtilBase(test: TableTestBase, isStreamingMode: Boolean)
 
   def doVerifyPlan(
       sql: String,
-      explainLevel: SqlExplainLevel,
       withRowType: Boolean,
       printPlanBefore: Boolean): Unit = {
     doVerifyPlan(
       sql = sql,
-      explainLevel = explainLevel,
       withRetractTraits = false,
       withRowType = withRowType,
       printPlanBefore = printPlanBefore)
   }
 
   def doVerifyPlan(
       sql: String,
-      explainLevel: SqlExplainLevel,
```

Comment:
please do not delete the parameter, it's used for testing different kinds plan, such as plan with cost (maybe those tests have not been ported until now)